sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Release History
 
-## 12.1.0-preview.1 (Unreleased)
+## 13.0.0
```

Comment:
@chidozieononiwu should we have the (Unreleased) moniker here?

Reply:
Yes, @Alex  You are right. One can just run the Update-Changelog script
> To set version it with Unreleased moniker. In most cases the build pipeline should handle this.
`& "eng/common/Update-Change-Log.ps1" -Version '13.0.0' -ChangeLogPath 'sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md'`

> To set version when preparing for release, with the current date in the header
`& "eng/common/Update-Change-Log.ps1" -Version '13.0.0' -ChangeLogPath 'sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md' -Unreleased false -ReplaceVersion true` to set the  current date in the changelog header

Reply:
Does the tooling automatically fix that once it goes live? If so, cool!

Reply:
After a release there will be a PR to update to the next version. But you should run it before releasing so that it sets the date. Or you can also add the date manually. just ,make sure you stick to the [changelog guidance](https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#changelog-guidance)

Reply:
I fixed this.